wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/privacy.html

```diff
@@ -4,21 +4,35 @@
 {% block content %}
     {% if page.id %}{% test_page_is_public page as is_public %}{% else %}{% test_page_is_public parent_page as is_public %}{% endif %}
 
-    {# The swap between public and private text is done using JS inside of privacy-switch.js when the response from the modal comes back #}
-    <div class="{% if not is_public %}w-hidden{% endif %}" data-privacy-sidebar-public>
-        {% trans 'Page visibility: ' as screen_reader_title_prefix %}
-        {% trans 'Visible to all' as title %}
-        {% trans 'Once live anyone can view' as help_text %}
-        {% with icon_name='view' %}
-            {{ block.super }}
-        {% endwith %}
-    </div>
-    <div class="{% if is_public %}w-hidden{% endif %}" data-privacy-sidebar-private>
-        {% trans 'Private' as title %}
-        {% trans 'Not visible to the public' as help_text %}
-        {% with icon_name='no-view' %}
-            {{ block.super }}
-        {% endwith %}
+    <div>
+        {# The swap between public and private text is done using JS inside of privacy-switch.js when the response from the modal comes back #}
+        <div
+            class="{% if not is_public %}w-hidden{% endif %}"
+            data-controller="w-zone"
+            data-action="w-privacy:changed@document->w-zone#switch"
+            data-w-zone-inactive-class="w-hidden"
+            data-w-zone-mode-value="{% if is_public %}active{% endif %}"
```

Comment:
Just a thought, we could try to make the `ZoneController` read the `this.element.className` values on connect, and then set the mode if the appropriate classes exist.

This would save us double declaring the 'mode' (in the initial classes and in the `mode-value`.